clients/admin-ui/src/features/integrations/configure-monitor/MonitorConfigEnableCell.tsx

```diff
@@ -0,0 +1,34 @@
+import { CellContext } from "@tanstack/react-table";
+
+import { EnableCell } from "~/features/common/table/v2/cells";
+import { usePutDiscoveryMonitorMutation } from "~/features/data-discovery-and-detection/discovery-detection.slice";
+import { MonitorConfig } from "~/types/api";
+
+const MODAL_TITLE = "Disabling monitor";
+const MODAL_TEXT =
+  "You are about to disable this monitor. If you continue, it will no longer scan automatically.";
+
+export const MonitorConfigEnableCell = ({
+  row,
+  getValue,
+}: CellContext<MonitorConfig, boolean | undefined>) => {
+  const [putMonitor] = usePutDiscoveryMonitorMutation();
+
+  const onToggle = async (toggleValue: boolean) =>
+    putMonitor({
+      ...row.original,
+      last_monitored: undefined,
```

Comment:
`last_monitored` is read-only and (doesn't currently but) will throw an error if it's sent to the backend.  See [here](https://ethyca.atlassian.net/browse/PROD-2145?focusedCommentId=41803) for context.